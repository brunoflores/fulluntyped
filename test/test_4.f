x = true;
x;
if x then false else x;
