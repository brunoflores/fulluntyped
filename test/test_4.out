: x: Bool
true : Bool
false
  : Bool
